### PR TITLE
Add varcode to Travis

### PR DESCRIPTION
**Pull request by tavinathanson**
And fix some linter errors along the way.

[Review on Reviewable](https://reviewable.io/reviews/hammerlab/varcode/120)

### CODE REVIEW

File: test/test_common.py [hunk 1]
```diff
@@ -59,3 +59,10 @@ def __repr__(self):
     eq_(tuple(sorted(grouped_dict.keys())), (1, 10))
     eq_(grouped_dict[1], [r1_2, r1_3])
     eq_(grouped_dict[10], [r10_20])
+
```
Comment (iskandr) [inline test/test_common.py]: Lots of newlines?

Comment (tavinathanson) [inline test/test_common.py]: I thought I was removing newlines; weird!

File: varcode/common.py [hunk 2]
```diff
@@ -48,3 +49,10 @@ def wrapped_fn(*args, **kwargs):
             return memoized_values[key]
 
     return wrapped_fn
+
+
+def read_csv_as_df(*args, **kwargs):
```
Comment (iskandr) [inline varcode/common.py]: Maybe unwise to redefine functions for pylint?

Comment (tavinathanson) [inline varcode/common.py]: You make a fair point. That said, I couldn't find a good way to disable pylint's `E: 68,11: Instance of 'tuple' has no 'columns' member (no-member)`-type errors. So I'm just disabling them manually, for now.

**Comment by tavinathanson** [comment]
@iskandr I also added coveralls. Back to you.

**Comment by iskandr** [comment]
LGTM

**Pull request merged**